src/LibTeXPrintf.jl

```diff
@@ -118,12 +118,11 @@ ArgumentError("\"not_a_font\" not in (\"mathsfbfit\", \"mathsfbf\", \"mathfrak\"
 ```
 """
 function texsetfont(font::String)
-    if font ∉ texfonts()
+    idx = findfirst(==(font), TEXFONTS)
```

Comment:
If you sort `TEXFONTS` (`const TEXTFONTS = sort!([...])`), then you could use `searchsortedfirst`